test/integration/vitesscluster/topology_test.go

```diff
@@ -113,13 +113,9 @@ func populateTopo(f *framework.Fixture, ts *topo.Server) {
 		f.Fatalf("Can't create keyspace: %v", err)
 	}
 
-	// To test shard and tablet pruning, we have to first create the actual
-	// desired keyspace and desired shards (which won't be pruned) because the
-	// operator doesn't do that. Normally the vttablets do that, but Vitess is
-	// not actually running here.
-	if err := ts.CreateKeyspace(f.Context(), "keyspace1", &topodatapb.Keyspace{}); err != nil {
-		f.Fatalf("Can't create keyspace: %v", err)
-	}
```

Comment:
Why did this get removed?

Reply:
Because it had encoded the bug in the test. If you notice the comment:
```
	// To test shard and tablet pruning, we have to first create the actual
	// desired keyspace and desired shards (which won't be pruned) because the
	// operator doesn't do that. Normally the vttablets do that, but Vitess is
	// not actually running here.
```

It was creating the keyspace *because the operator did not create it*. That was the bug. So when we fixed that bug, this test failed with a topo `NodeExists` error when trying to create the keyspace because the keyspace already existed. Once we removed this block which was added to work around the bug, then the test passed.